samples/ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor/ChangeFeedEventHost.cs

```diff
@@ -476,7 +476,7 @@ static DocumentCollectionInfo CanoninicalizeCollectionInfo(DocumentCollectionInf
             this.collectionSelfLink = collection.SelfLink;
 
             // Grab the options-supplied prefix if present otherwise leave it empty.
-            string optionsPrefix = string.IsNullOrWhiteSpace(this.options.LeasePrefix) ? string.Empty : this.options.LeasePrefix + "_";
+            string optionsPrefix = string.IsNullOrWhiteSpace(this.options.LeasePrefix) ? string.Empty : this.options.LeasePrefix;
```

Comment:
How's the latest? (string optionsPrefix = this.options.LeasePrefix ?? string.Empty;)